src/databases/index.js

```diff
@@ -55,4 +56,9 @@ const getDatabaseType = (type) => {
   return databaseTypes[type]
 }
 
+addDatabaseType(Events)
+addDatabaseType(Documents)
+addDatabaseType(KeyValue)
+addDatabaseType(KeyValueIndexed)
+
```

Comment:
Noticing this only now but there prolly should be `removeDatabaseType` also so that the user can remove a default handler and add their custom handler for any database type.

Reply:
Removing removeX and using useX (as discussed).